src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Figure implements Area, Output{
```

Comment:
Don't use verbs for class/interface names. The interface 'Area' should be named following the noun convention, such as 'AreaCalculator'.